Assets/MRTK/SDK/Features/UX/Scripts/Pointers/SpherePointer.cs

```diff
@@ -108,8 +141,13 @@ public bool IgnoreCollidersNotInFOV
 
         private void Awake()
         {
-            queryBufferNearObjectRadius = new SpherePointerQueryInfo(sceneQueryBufferSize, NearObjectRadius);
-            queryBufferInteractionRadius = new SpherePointerQueryInfo(sceneQueryBufferSize, SphereCastRadius);
+            InitQueryParameters();
+        }
+
+        public void InitQueryParameters()
+        {
+            queryBufferNearObjectRadius = new SpherePointerQueryInfo(sceneQueryBufferSize, NearObjectRadius, NearObjectSectorAngle, PullbackDistance);
```

Comment:
For the near object check, seems like we should also have the min distance parameter? To cover removing the 'tip' of the cone.

Reply:
Based on the implementation in the HoloLens shell I thought this was the same as the pullback distance. You can see a minimum distance check inside the `TryUpdateQueryBufferForLayerMask` function

Reply:
Ah okay, gotcha.